packages/in_app_purchase/lib/src/billing_client_wrappers/billing_client_wrapper.dart

```diff
@@ -172,11 +172,17 @@ class BillingClient {
   /// and [the given
   /// accountId](https://developer.android.com/reference/com/android/billingclient/api/BillingFlowParams.Builder.html#setAccountId(java.lang.String)).
   Future<BillingResultWrapper> launchBillingFlow(
-      {@required String sku, String accountId}) async {
+      {@required String sku,
+      String accountId,
+      String oldSku,
```

Comment:
Can we add dart doc to explain the 2 new parameters?

Reply:
Done